tasks/hardening.yml

```diff
@@ -98,12 +98,19 @@
   include_tasks: selinux.yml
   when: ansible_facts.selinux and ansible_facts.selinux.status == "enabled"
 
-- name: disable system CRYPTO_POLICY for RHEL8+
+- name: check SSH server CRYPTO_POLICY
+  find:
+    path: /etc/sysconfig
+    pattern: sshd
+    contains: '.*CRYPTO_POLICY=.*'
+  register: crypto_policy
```

Comment:
I found no better way of detecting if we have to disable the CRYPTO_POLICY, that also works with all supported OSes

Considered alternatives were:
* package_facts_module search for package `crypto-policies`
Possible alternative, but maybe more ressource intensive. Also needs extra task but no register
* OS version check
Would need to construct a check which covers CentOS/RHEL/Oracle and Fedora with differend release versions, that would be very complicated
* some varaible from default facts
Nothing of value found there